webapp/src/services/ResponsePage/ResponsePage.js

```diff
@@ -0,0 +1,33 @@
+
+const data = [
```

Comment:
Please name this file consistently with the other services (eg `responseDetail.service.js`) and the directory name in lower case, something like "responseDetail"